mautic/mautic
# Update grapesjs and other js dependencies

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ partly]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 


#### Description:

Updates the Grapesjs dependencies and some other smaller dev dependencies:
https://github.com/artf/grapesjs/releases
https://github.com/artf/grapesjs-mjml/releases

All updates:

 grapesjs                                 ^0.17.22  →  ^0.17.29     
 grapesjs-mjml                              ^0.5.5  →    ^0.5.8     
 @babel/cli                                ^7.14.8  →   ^7.16.8     
 @babel/core                               ^7.15.0  →   ^7.16.7     
 @babel/plugin-proposal-class-properties   ^7.14.5  →   ^7.16.7     
 @babel/plugin-transform-runtime           ^7.15.0  →   ^7.16.8     
 eslint                                    ^7.32.0  →    ^8.7.0     
 eslint-config-airbnb                      ^18.2.1  →   ^19.0.4     
 eslint-plugin-import                      ^2.24.1  →   ^2.25.4     
 eslint-plugin-jsx-a11y                     ^6.4.1  →    ^6.5.1     
 eslint-plugin-prettier                     ^3.4.1  →    ^4.0.0     
 eslint-plugin-react                       ^7.24.0  →   ^7.28.0     
 eslint-plugin-react-hooks                  ^4.2.0  →    ^4.3.0     
 prettier                                   ^2.3.2  →    ^2.5.1     
 sass                                      ^1.38.1  →   ^1.48.0 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. See if the e-mail builder works as expected (mjml and HTML mode)
3. See if landing page builder works as expected



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10760"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
